protobuf/protobuf_test.go

```diff
@@ -52,3 +54,57 @@ func TestOptionsString(t *testing.T) {
 	require.Equal(t, "foo", NewLiteralValue("foo").String())
 	require.Equal(t, `"bar"`, NewStringValue("bar").String())
 }
+
+func TestIsNullable(t *testing.T) {
+	var typ Type
```

Comment:
Because the logic is not the same, some depend uniquely on the source, some does not, etc.